common/hugo/hugo_test.go

```diff
@@ -30,7 +30,7 @@ func TestHugoInfo(t *testing.T) {
 	c.Assert(hugoInfo.CommitHash, qt.Equals, commitHash)
 	c.Assert(hugoInfo.BuildDate, qt.Equals, buildDate)
 	c.Assert(hugoInfo.Environment, qt.Equals, "production")
-	c.Assert(string(hugoInfo.Generator()), qt.Contains, fmt.Sprintf("Hugo %s", hugoInfo.Version()))
+	c.Assert(string(hugoInfo.Generator()), qt.Contains, fmt.Sprintf("Gotham %s", hugoInfo.Version()))
```

Comment:
Does this pass?

Reply:
No this does not pass because hugoInfo is based on CurrentVersion.Version()

Reply:
Instead of `hugoInfo.Version()` we'd use `GothamVersion.String()`.

Also, a couples up there's a related test that's checking the type. That test we can remove. Our version string is just a string, not a custom type.